request_a_govuk_domain/request/templates/base.html

```diff
@@ -66,18 +66,20 @@
 
   {% block phase_banner %}
     <div class="govuk-phase-banner govuk-width-container">
-      <p class="govuk-phase-banner__content">
+      <div class="govuk-phase-banner__content">
         <strong class="govuk-tag {{ PHASE_CLASS }} govuk-phase-banner__content__tag">
           {% block phase_banner__tag %}
               {{PHASE_HEADER}}
           {% endblock %}
         </strong>
         <span class="govuk-phase-banner__text">
           {% block phase_banner__text %}
+          {% autoescape off %}
```

Comment:
Why did you add this? Can you explain in the commit message?
Also: fix indentation

Reply:
> Why did you add this? Can you explain in the commit message? Also: fix indentation
Done, fixed indentation and added the autoescape explanation.

Reply:
> Why did you add this? Can you explain in the commit message? Also: fix indentation

Done, fixed Indentation and added the autoescape explanation.